src/PanelTraits/Create.php

```diff
@@ -21,14 +21,22 @@ public function create($data)
     {
         $data = $this->decodeJsonCastedAttributes($data, 'create');
         $data = $this->compactFakeFields($data, 'create');
-
+        
         // ommit the n-n relationships when updating the eloquent item
         $nn_relationships = array_pluck($this->getRelationFieldsWithPivot('create'), 'name');
-        $item = $this->model->create(array_except($data, $nn_relationships));
-
+        $after_create = array_pluck($this->getAfterCreateFields(), 'name');
+        
+        $creationData = array_except($data, array_merge($nn_relationships, $after_create));
+        $item = $this->model->create($creationData);
+        
+        $afterCreateData = array_only($data, $after_create);
+        foreach ($afterCreateData as $key => $value) {
+            $item->{$key} = $value;
```

Comment:
Is it possible for `$value` to come up with `isset() !== true` and trigger an undefined key warning/error?